src/redux/books/books.js

```diff
@@ -32,7 +32,7 @@ export const fetchBooksSuccess = (payload) => ({
   payload,
 });
 export const fetchBooks = () => async (dispatch) => {
-  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HmsXokvT4r0o4d4gM6bR/books');
+  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SERz0T6MOlfncfi0umcc/books');
   const bookData = await response.json();
   const books = Object.entries(bookData).map(([key, value]) => ({
     item_id: key,
```

Comment:
Because new books fail to be saved to the API, the fetched data is always empty. Therefore, when I refresh the page all the data is lost because the state is cleared. After refreshing the page, the books should not disappear since they are saved externally and therefore cannot be lost unless deleted.

![image](https://user-images.githubusercontent.com/38088818/166979135-df225cef-3936-45bc-8930-1db6c7f0ebc8.png)

- Please click on the image to see the comments clearly.